src/fondue/Fondue.js

```diff
@@ -760,10 +794,112 @@ export default class Fondue {
 				});
 			}
 
-			return currentAllGlyphs;
+			// Chained context substitution
+			// Note that currently if a ChainContextSubst contains multiple coverages,
+			// we merge them all into one and remove duplicates. This is purely to keep
+			// a "data dump" less overwhelming, but isn't a perfect treatment of a type
+			// 6 lookup, as this can suggest character combinations that aren't
+			// possible.
+			// Possible future improvement: add the chars as an array to an array of
+			// input/backtrack/lookahead, then let a front decide whether to merge
+			// them or dump everything as-is.
+
+			// TODO: When a lookup has a lookahead or backtrack with *only* glyphs (so
+			// no direct characters), we should ignore this lookup. Otherwise it will
+			// result in a lookup that *looks* okay, but isn't.
+			// Example: backtrack [a, b, c], input [n], lookahead[x.alt, y.alt, z.alt]
+			// Since the lookhead contains no chars, it will be reduced to [], and the
+			// lookup with look like this: backtrack [a, b, c], input [n]. This is
+			// wrong, as that only the backtrack+input will not result in any changed
+			// chars
+			if (lookup.lookupType === 6) {
+				lookup.subtableOffsets.forEach((_, i) => {
+					const subtable = lookup.getSubTable(i);
+
+					let inputChars;
+					let backtrackChars;
+					let lookaheadChars;
+
+					if (subtable.inputGlyphCount > 0) {
+						subtable.inputCoverageOffsets.forEach((offset) => {
+							const coverage = subtable.getCoverageFromOffset(
+								offset
+							);
+							inputChars = charactersFromGlyphs(coverage);
```

Comment:
This lambda will get executed multiple times potentially, but it always overwrites the `inputChars` variable. Is that intentional? E.g. does only the last offset matter? Same for all lookups below.